blueprintapi/oscal/oscal.py

```diff
@@ -271,7 +271,7 @@ class RoleIDEnum(str, Enum):
 
 
 class Role(OSCALElement):
-    id: RoleIDEnum
+    id: str
```

Comment:
So then is it no longer expected that the `id` will be one of `RoleIDEnum`?

Reply:
It turns out that Roles defined in that enum are FedRAMP roles, but not necessarily universally specific.

From the OSCAL Gitter channel:
`Since <role> lacks any means of association (has no suitable attribute) with a well-known framework (such as NIST RMF roles defined in NIST SP 800-37 Appendix D) the role identifier is essentially unconstrained.`